wrappers/pascal/README.md

```diff
@@ -31,27 +30,28 @@ You need a modern Pascal compiler. You have two options:
 
 
 ### Lazarus
-The IntelRealsense is a 64 bits library. So you must create a 64 bits aplication. 
 
 If you don't have a copy of lazarus download Lazarus for 64 bits from here: https://www.lazarus-ide.org/index.php?page=downloads
 
-Be sure that you download the correct version.
+Make sure you download the correct version for your operating system.
 ![Lazarus 64 bits](./images/installlazarus64.png)
 
 Download or clone the repository in a folder where you like.
 
 From Lazarus, open a package.
 ![Open pacakge](./images/lazarusopenpackage.png)
```

Comment:
I think if we add one empty line before a picture, it will appear on the right side every time and not in the middle of a page. See the attached screenshot that the picture posited in the middle of the page right after the text.
![image](https://user-images.githubusercontent.com/26552140/228846220-1df26ed1-59b6-406e-8aa9-b2719d2d1077.png)